cctp/domains.json

```diff
@@ -0,0 +1,8 @@
+[
+    "0",
+    "1",
+    "2",
+    "3",
+    "6",
+    "7"
+]
```

Comment:
No need for this file. It's not required

Reply:
removed